catalog_tools/plots/basics.py

```diff
@@ -34,7 +54,7 @@ def plot_cum_fmd(ax: plt.Axes, mags: np.ndarray, color: str = 'blue',
         if mc is None:
             mc = min(mags)
         x = mags[mags >= mc]
-        y = len(x) * 10 ** (-b_value * (x - mc))
+        y = gutenberg_richter(x, b_value, mc, len(x))
```

Comment:
why is n_mc equal to len(x) here? probably all good, I am probably just confused. if this is always the case, then the argument of n_mc is not needed.

Reply:
This happens only if you don't give an mc but a b-value to fit. Then it will assume that the magnitudes are precut, but it also makes sense to be able to tell it the mc